### PR TITLE
fix notifications

### PR DESCRIPTION
**Pull request by dikhomenko**

### CODE REVIEW

File: src/js/footer.js [hunk 1]
```diff
@@ -1,73 +1,76 @@
...
+      message: 'Please enter a valid email address.',
     });
     return;
   }
 
   const subscriptionData = { email };
 
-  fetch("https://your-energy.b.goit.study/api/subscription", {
-    method: "POST",
+  fetch('https://your-energy.b.goit.study/api/subscription', {
```
Comment (OleinikovArtem) [inline src/js/footer.js]: use `axios` instead

File: src/partials/footer.html [hunk 2]
```diff
@@ -38,18 +38,29 @@
...
+        >Terms of Service</a
+      >
     </div>
   </div>
   <div class="footer-bottom">
     <p>Your Energy. All rights reserved. ©2024</p>
   </div>
-  <script src="/js/footer.js" type="module"></script>
+  <script src="../js/footer.js" type="module"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/izitoast/1.4.0/js/iziToast.min.js"></script>
```
Comment (OleinikovArtem) [inline src/partials/footer.html]: move it to main index.html file

Comment (dikhomenko) [reply src/partials/footer.html]: done

File: src/js/footer.js (hunk 1 above)
Comment (dikhomenko) [reply src/js/footer.js]: done

**Review by OleinikovArtem — APPROVED** [review]

**Pull request merged**